app/views/site/covid19.html.erb

```diff
@@ -61,8 +61,6 @@
   </div>
 </div>
```

Comment:
Ah, yup, this was my fault. But we should update this to match `application.html.erb` rather than deleting it.  that will keep things in sync between the searches on the covid space and the regular home page
```suggestion
</div>

<input type="hidden" id="user-provider-info" value='<%=
  {
    email: current_user&.email,
    featureFlagsWithDefaults: FeatureFlaggable.feature_flags_for_instances(@selected_branding_group, current_user)
  }.to_json %>'/>
```